client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/RemoteQueryStringTest.java

```diff
@@ -74,9 +74,9 @@ public void registerMarshallers(SerializationContext serCtx) {
    protected void populateCache() throws Exception {
       super.populateCache();
 
-      getCacheForWrite().put("analyzed1", new AnalyzerTestEntity("tested 123", 3));
+     /* getCacheForWrite().put("analyzed1", new AnalyzerTestEntity("tested 123", 3));
       getCacheForWrite().put("analyzed2", new AnalyzerTestEntity("testing 1234", 3));
-      getCacheForWrite().put("analyzed3", new AnalyzerTestEntity("xyz", null));
+      getCacheForWrite().put("analyzed3", new AnalyzerTestEntity("xyz", null));*/
```

Comment:
I uncommented those lines and the test passes, is this a leftover?

Reply:
Yes it is. I seems that with the last rebase of the test, now the serialization context contains all the protobuf definitions since the beginning (at cache start in `remote.LifecycleManager`). I’m going to uncomment the lines.